lib/flutter_barcode_listener.dart

```diff
@@ -78,15 +79,30 @@ class _BarcodeKeyboardListenerState extends State<BarcodeKeyboardListener> {
     //remove any pending characters older than bufferDuration value
     checkPendingCharCodesToClear();
     _lastScannedCharCodeTime = DateTime.now();
+    callWithoutLineFeed();
```

Comment:
I have problem with this, while this additional call will probably workaround Keyence issue, it will be called for all users, even when great majority of them does not have this problem.